lib/text/text.js

```diff
@@ -45,6 +45,15 @@ export default class OCText extends React.Component {
         disabled: this.props.disabled
       });
     }
+
+    if (
+      this.props.value !== previousProps.value ||
+      this.props.value !== this.state.value
+    ) {
+      this.setState({
```

Comment:
Hi @Upendragorrepati setstate inside componenetdidupdate is not advisible. as it creates an infinite loop.
https://stackoverflow.com/questions/30528348/setstate-inside-of-componentdidupdate

Please see other options to have this functionality.

Reply:
It creates an infinite loop If we do not use it properly...This is wrapping inside a condition. should be fine